flask_advanced/blueprint/Products/product.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from flask import Blueprint, render_template, request, session, url_for, redirect
+
+from flask_advanced.add_forms import ProductForm
+from flask_advanced.utils import add_data, get_data
+
+product = Blueprint('product', __name__, template_folder='templates', static_folder='static1')
+
+path_data_products = "blueprint/Products/data.json"
```

Comment:
if this variable is static and you never change it, name this as a constant like `I_AM_CONSTANT`